tests/lib/wast/src/wast.rs

```diff
@@ -171,6 +173,8 @@ impl Wast {
                 let result = self.perform_invoke(call);
                 self.assert_trap(result, message)?;
             }
+            #[cfg(feature = "test-no-traps")]
```

Comment:
Is there an issue filed either on us or on tarpaulin for this? Could you put a link to it in a comment here?